src/Lookup/LabelLookup.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Lookup;
+
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Term\Term;
+
+/**
+ * @since 3.9
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+interface LabelLookup {
+
+	/**
+	 * @param EntityId $entityId
+	 *
+	 * @throws LabelDescriptionLookupException
```

Comment:
This is nitpicking, but this exception is not totally accurate here :/

Reply:
Agreed

Reply:
I suggest to rename this exception to `TermLookupException` or something similar. Can we please do this later, in a separate patch?

Reply:
Sure thing :)